includes/wp-admin/import/class-mastodon.php

```diff
@@ -95,7 +95,7 @@ public static function handle_upload() {
 
 		\check_admin_referer( 'import-upload' );
 
-		if ( ! isset( $_FILES['import']['name'] ) ) {
+		if ( ! isset( $_FILES['import']['name'] ) || empty( $_FILES['import']['name'] ) ) {
```

Comment:
Empty implicitly checks for `isset`, so no need to have both.

```suggestion
		if ( empty( $_FILES['import']['name'] ) ) {
```